src/z80_macros.h

```diff
@@ -0,0 +1,96 @@
+/* Caprice32 - Amstrad CPC Emulator
+   (c) Copyright 1997-2004 Ulrich Doewich
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+*/
+
```

Comment:
Please add a TODO referencing issue #150 and explaining that we should get rid of the leading _ in the macros (either by renaming the macros or getting rid of them).